pkg/logger/configure.go

```diff
@@ -37,11 +47,82 @@ func ConfigureDevelopmentLogger(level string, syncs ...io.Writer) (Logger, error
 		zap.Fields(zap.String("version", Version)),
 	)
 	zap.ReplaceGlobals(zapLogger)
-	return &logger{underlyingLogger: zapLogger}, nil
+
+	traceExporter, err := newExporter(ctx)
+	if err != nil {
+		zap.L().Fatal("failed to create trace exporter", zap.Error(err))
+	}
+
+	tp := newTraceProvider(traceExporter)
+	otel.SetTracerProvider(tp)
+
+	l := &logger{underlyingLogger: zapLogger}
+	ctx = l.InjectIntoContext(ctx)
+	ctx = tracer.NewContext(ctx, tp, "dev-logger-tracer")
```

Comment:
should we make this conditional since this logger is used for the CLI tool as well

Reply:
yeah we dont want the tracing output to stdout in the CLI tool. make all of this opt in with an env var.

Reply:
@tim-thacker-nullify yeah this has been done with the latest commit